nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-kubernetes-bundle/nifi-framework-kubernetes-state-provider/pom.xml

```diff
@@ -48,5 +48,10 @@
             <artifactId>kubernetes-server-mock</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-expression-language</artifactId>
```

Comment:
Is this dependency required?

Reply:
It's needed for 

```java
when(context.getProperty(KubernetesConfigMapStateProvider.CONFIG_MAP_NAME_PREFIX))
                .thenReturn(new StandardPropertyValue(null, null, ParameterLookup.EMPTY));
```
and
```java
when(context.getProperty(KubernetesConfigMapStateProvider.CONFIG_MAP_NAME_PREFIX))
                .thenReturn(new StandardPropertyValue(CONFIG_MAP_NAME_PREFIX_VALUE, null, ParameterLookup.EMPTY));
```

For `StandardPropertyValue` and `ParameterLookup`.

Reply:
Thanks for the clarification, that makes sense given that `nifi-expression-language` is also a dependency of the more general `nifi-mock` test library. This looks good.